liberty-archetype-mp/src/test/resources/projects/it/archetype.properties

```diff
@@ -0,0 +1,10 @@
+groupId=test.wlp
+artifactId=ServletSample
+version=0.1-SNAPSHOT
+libertyGroupId=io.openliberty
+libertyArtifactId=openliberty-runtime
+wlpRuntimeVersion=17.0.0.4
+wlpPluginVersion=2.1.2
```

Comment:
Can we get this version dynamically from the archetype command?

Reply:
If I understand correctly, these versions are a statement of what's been tested/supported.  I took this straight from the other archetypes (which doesn't mean I'm suggesting it's right and shouldn't be changed).  Do you have any pointers for how to do this?

Reply:
I'm not sure. I think the other archetypes are being updated manually at the moment, so we can probably leave as is for now.

Reply:
I think leaving this for now and addressing it for all the archetypes is probably the right think to do.  I'm not sure what the best approach to updating them should be (yes, it is manual at the moment).